resources/src/main/java/org/robolectric/res/android/CppApkAssets.java

```diff
@@ -88,31 +88,31 @@ public LoadedArsc GetLoadedArsc() {
   Asset idmap_asset_;
   private LoadedArsc loaded_arsc_;
   // };
-//
-// }  // namespace android
-//
-// #endif /* APKASSETS_H_ */
-//
-// #define ATRACE_TAG ATRACE_TAG_RESOURCES
-//
-// #include "androidfw/ApkAssets.h"
-//
-// #include <algorithm>
-//
-// #include "android-base/logging.h"
-// #include "utils/FileMap.h"
-// #include "utils/Trace.h"
-// #include "ziparchive/zip_archive.h"
-//
-// #include "androidfw/Asset.h"
-// #include "androidfw/Util.h"
-//
-// namespace android {
-//
-// Creates an ApkAssets.
-// If `system` is true, the package is marked as a system package, and allows some functions to
-// filter out this package when computing what configurations/resources are available.
-// std::unique_ptr<const ApkAssets> ApkAssets::Load(const String& path, bool system) {
+  //
+  // }  // namespace android
+  //
+  // #endif // APKASSETS_H_
```

Comment:
It is copied from C++ source code, we don't need to change it. What about suppressing it?

Reply:
Sure

Reply:
https://github.com/robolectric/robolectric/pull/7354/files#diff-63779c53fd92fe4eea8da36ee86c8ee5035b73234e4818d9b95f0398ae5de984R94

Reply:
Emm. I think you can move SupressWarnings on top of Java method.